src/api/note-vdom.js

```diff
@@ -16,7 +16,7 @@ var hasAttribute = require('../utils/vdom/has-attribute');
 var isTag = require('../utils/vdom/is-tag');
 var isEmpty = require('../utils/vdom/is-empty');
 
-var focusOnNote = require('../utils/noting/is-note');
+var focusOnNote = require('../utils/noting/is-note-segment');
```

Comment:
They have now been changed. Doesn't make sense to change this one however as we will be removing this file soon :)